simplq/src/store/asyncActions/getQueueStatus.js

```diff
@@ -0,0 +1,30 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import useAuth, { makeAuthedRequest } from 'api/auth';
+import * as RequestFactory from 'api/requestFactory';
+
+const typePrefix = 'getQueueStatus/requestStatus';
+
+/**
+ * A hook to access the deleteQueue async action creator.
```

Comment:
it should be `getQueueStatus async action creator`.

Should we be naming this `getQueueStatusById`?

Reply:
In some other language, I would probably just overload the `getQueueStatus(queueID)`. Let's keep it as is for now.